sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -367,7 +367,16 @@ private case class OrcPartitionReaderContext(
     dataReader: DataReader,
     orcReader: Reader,
     blockIterator: BufferedIterator[OrcOutputStripe],
-    requestedMapping: Option[Array[Int]])
+    requestedMapping: Option[Array[Int]]) {
+  private var isClosed = false
+
+  def cleanUp = {
```

Comment:
Nit: Lack of parentheses should be reserved for methods that have no side effects.  Since this mutates the state of the object, it should have parentheses and be called with parentheses.
```suggestion
  def cleanUp() = {
```

Reply:
Thx Jason for the comment. Done